test/unit/connectors/CustomsDeclareExportsMovementsConnectorSpec.scala

```diff
@@ -204,4 +205,29 @@ class CustomsDeclareExportsMovementsConnectorSpec extends ConnectorSpec with Moc
       response mustBe Seq(expectedNotification)
     }
   }
+
+  "Submit ILE Query" should {
+
+    "POST to the Back End" in {
+      stubFor(
+        post("/consignment-query")
+          .willReturn(
+            aResponse()
+              .withStatus(Status.ACCEPTED)
+              .withBody(s"$conversationId")
+              .withHeader("Content-Type", "application/json")
+          )
+      )
+
+      val request = IleQueryRequest("eori", Some("provider-id"), UcrBlock("ucr", "D"))
+      val result = connector.submit(request).futureValue
+
+      verify(
+        postRequestedFor(urlEqualTo("/consignment-query"))
+          .withRequestBody(equalToJson("""{"eori":"eori","providerId":"provider-id","ucrBlock":{"ucr":"ucr","ucrType":"D"}}""".stripMargin))
+      )
+
+      result mustBe conversationId
+    }
+  }
```

Comment:
Nit: Could you move this test to be next to other submission tests?

Reply:
Fixed